apps/test/unit/applab/exportTest.js

```diff
@@ -0,0 +1,29 @@
+import {assert} from '../../util/configuredChai';
+import {blocks} from '@cdo/apps/applab/dropletConfig';
+import {getExportedGlobals} from '@cdo/apps/applab/export';
+
+describe("the export module", () => {
+  describe("the getExportedGlobals function", () => {
+    let globals;
+    before(() => {
+      globals = Object.keys(getExportedGlobals());
+    });
+    blocks.forEach(block => {
+      if (block.blockPrefix || block.block) {
+        // we don't care about functions defined on objects other than window
+        // for example: Array.indexOf. We also don't care about this with blocks,
+        // like declareAssign_str_hello_world, which is just:
+        //
+        //   var str = "Hello World";
+        return;
+      }
+      it(`exposes a ${block.func} function`, () => {
+        if (globals.indexOf(block.func) < 0) {
+          if (!eval(`window.${block.func}`)) { //eslint-disable-line no-eval
+            assert(false, `expected exported globals to include ${block.func}`);
+          }
+        }
+      });
```

Comment:
exported apps do not run through our interpreter. Instead, you just end up with a script tag pointing to a file with the student's js code. As a result, and functions that the student's code might call to need to be on `window`.